index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Personal Portfolio</title>
+  <link rel="stylesheet" href="main.css" />
+ 
+</head>
+
+	<body>
+		<header class="Flexbox-header">
+			<div
+				class="flex-container header flex-row align-center justify-between"
+			>
+				<a href="#" class="logo color-primary">My Logo</a>
+				<svg
+					class="NavToggle"
+					width="24"
+					height="24"
+					viewBox="0 0 24 24"
+					fill="none"
+					xmlns="http://www.w3.org/2000/svg"
+				>
```

Comment:
- [OPTIONAL] You have done a great job here, in implementing the changes requested by the Previous Code reviewer, I kindly suggest that you please wrap the Hamburger menu in a `<nav>` tag.  Wrapping a hamburger menu in a `<nav>` tag in HTML is generally considered a good practice and it is also part of the requirement. The `<nav>` element is semantically appropriate for representing a section of a webpage that contains navigation links, such as a menu. While the specific visual representation of the menu (e.g., hamburger icon) may vary depending on the CSS and JavaScript implementation, using the `<nav>` element helps convey the intended purpose of the content to assistive technologies, search engines, and other user agents. :slightly_smiling_face: